core/src/banking_stage/transaction_scheduler/receive_and_buffer.rs

```diff
@@ -430,6 +430,7 @@ impl TransactionViewReceiveAndBuffer {
                     if result.is_err() {
                         num_dropped_on_status_age_checks += 1;
```

Comment:
(github won't let me comment at line 444 - 445)
1. at ln 444, `num_dropped_on_status_age_checks += 1;` is misleading as the drop due to fee-payer
2. maybe safer to add `continue` after ln 445 `container.remove_by_id(priority_id.id);`, in case in the future code being added after fee_payer_check

Reply:
08ca3aaf25